[1,"Error occurred during cleanup: Error: The process cannot access the file because it is being used by another process. (os error 32): remove 'C:\\Users\\user\\AppData\\Local\\Temp\\quarto-sessionf49d594940488ec9'\n"]
